packages/react-scripts/README.md

```diff
@@ -23,6 +23,7 @@ npm start
   - `crossOriginLoading`: Modify the default behaviour, see [docs](https://webpack.js.org/configuration/output/#output-crossoriginloading).
   - `babelIncludePrefixes`: An array of module name prefixes to opt into babel compilation. Includes `["bpk-", "saddlebag-"]` by default.
   - `enableAutomaticChunking`: Boolean, opt in to automatic chunking of vendor, common and app code.
+  - `overrideChunkingConfig`: Object, config to override specific chunking groups. See `cacheGroups` in webpack docs. e.g. Allows regex test for chunking certain packages
```

Comment:
After thinking about this further, I don't want to open this up too much. How about we follow previous examples and only allow prefixes to be configured for the vendor chunk? 
Take `bableIncludePrefixes` as an example:
```
- `babelIncludePrefixes`: An array of module name prefixes to opt into babel compilation. Includes `["bpk-", "saddlebag-"]` by default.
```

At the very least we should always have `bpk-` prefixes excluded by default. As for a name ...`vendorChunkExcludePrefixes`?

Reply:
Also we should mention that this is dependent on `enableAutomaticChunking `

Reply:
Regarding this one, what is the issue with opening this up?
And also only allowing prefixes, that would stop the ability to work the other way round. i.e. choose to only use certain packages in vendors rather than exclude them. The idea to only exclude certain packages is restricting depending on the project

Reply:
Yup good question. It's about trying to maintain the "zero-configuration" aspect of `create-react-app`.

The reason they have been so successful in making future upgrades easy for consumer is down to the lack of configuration and API's that it exposes. Essentially all you need to do is expose an `index.js` and a `public/index.html` and it "just works".

To bring it back to this example - what happens if webpack changes the api of `cacheGroups`? that would force a breaking change for all consumers. IF we had `vendorChunkExcludePrefixes` then it's more likely that we'll be able to handle the breaking change in the BRS upgrade itself and maintain backwards compat for consumers.

In terms of flexibility - I would tend to lean on what CRA's functionality currently is but with some tweaks to make sure our Skyscanner specific (mostly backpack) still work with it.

Reply:
Actually ... perhaps we could just hard code `(bpk-|banana-)` and be done with it. Especially for the `pre-release` version that we wanna "test".

Reply:
Fair enough. I get that logic. How about we just pass in a regex for vendorsChunk? This would keep the functionality open enough to pick and choose what goes into vendors chunk. We could call it `vendorsChunkRegex`

Reply:
Yeah ok let's go with `vendorsChunkRegex`.